parachain/runtime/interlay/src/evm.rs

```diff
@@ -0,0 +1,95 @@
+use crate::{
+    AccountId, Aura, BaseFee, EVMChainId, NativeCurrency, Runtime, RuntimeEvent, Timestamp, CENTS,
+    MAXIMUM_BLOCK_WEIGHT, NORMAL_DISPATCH_RATIO,
+};
+use frame_support::{
+    parameter_types,
+    traits::{ConstU32, FindAuthor},
+    weights::Weight,
+    ConsensusEngineId,
+};
+use pallet_ethereum::PostLogContent;
+use pallet_evm::{EnsureAddressRoot, EnsureAddressTruncated, FixedGasWeightMapping, HashedAddressMapping};
+use sp_core::{crypto::ByteArray, H160, U256};
+use sp_runtime::{traits::BlakeTwo256, Permill};
+use sp_std::marker::PhantomData;
+
+pub use runtime_common::evm::{
+    precompiles::InterBtcPrecompiles, BaseFeeThreshold, GetTransactionAction, SetEvmChainId, WEIGHT_PER_GAS,
+};
+
+pub type Precompiles = InterBtcPrecompiles<Runtime>;
+pub type AccountConverter = HashedAddressMapping<BlakeTwo256>;
+
+parameter_types! {
+    pub DefaultBaseFeePerGas: U256 = U256::from(CENTS);
```

Comment:
The "flipper" contract fixture costs 133751 gas to create which would total `(133751 * (10**10 / 100)) / 10**10 = 1337.51 INTR ($19.7)`.